docs/source/protocol_v3.rst

```diff
@@ -1,51 +1,437 @@
-Proposed Protocol v3
-====================
+IMDv3 Protocol
+==============
 
-The suggested changes to the protocol are as follows:
+IMDv3 is a protocol for communicating molecular simulation data through a socket. 
+It allows for two-way communication: via IMDv3, a simulation engine sends data to a receiver 
+and the receiver can send forces and certain requests back to the simulation engine.
 
-1. Via an ``.mdp`` file setting, the user should be able to specify which of simulation box, positions, forces, and velocities are sent.
-   These should be rate-adjustable like with xtc and trr output settings but should be set to every step by default.
+Terms
+-----
+**Simulation engine**: The producer of simulation data which listens for a receiver connection and sends it simulation data
 
-2. The IMD_HANDSHAKE packet should have a 1-byte body which contains the configuration settings the simulation was setup with.
-   This will allow the client to be choose appropriate packets to send and receive without redundant configuration.
+**Receiver**: The receiver of simulation data, connects to listening simulation engine and receives simulation data through a socket
 
-   The modified handshake packet would look like this:
+**IMD frame**: A simulation frame in which IMD data will be sent from the simulation engine to the receiver
```

Comment:
Using frame  in the explanation is a bit tautological. Maybe “data structure”?

Reply:
Is "integration step" appropriate? Don't want to use "data structure" since the struct only exists on the receiver side and I'm using IMD frame in this doc to simplify talking about IMD output without constantly mentioning the transmission rate

Reply:
sounds reasonable to me as written in the new version